vars/AlmaLinux_6.yml

```diff
@@ -0,0 +1 @@
+RedHat_6.yml
```

Comment:
I assume this is mostly mechanical edit, but do Alma and Rocky even build EL6 and EL7? On the website I see only 8 and 9 for Rocky: https://rockylinux.org/download
I would like to avoid introducing bogus configuration like that, that is not useful for anything ...

Reply:
> I assume this is mostly mechanical edit, but do Alma and Rocky even build EL6 and EL7? On the website I see only 8 and 9 for Rocky: https://rockylinux.org/download I would like to avoid introducing bogus configuration like that, that is not useful for anything ...

Ok.  I can change the automation https://github.com/linux-system-roles/.github/pull/74 to skip version < 8 when distribution is AlmaLinux or Rocky.

Reply:
done